src/annotator/annotation-counts.js

```diff
@@ -7,11 +7,11 @@ const ANNOTATION_COUNT_ATTR = 'data-hypothesis-annotation-count';
  *
  * @param {Element} rootEl - The DOM element which contains the elements that
  *   display annotation count.
- * @param {import('../shared/bridge').Bridge} bridge - Channel for host-sidebar communication
+ * @param {import('../shared/port-rpc').PortRPC} rpc - Channel for host-sidebar communication
  */
 
-export function annotationCounts(rootEl, bridge) {
-  bridge.on('publicAnnotationCountChanged', updateAnnotationCountElems);
+export function annotationCounts(rootEl, rpc) {
+  rpc.on('publicAnnotationCountChanged', updateAnnotationCountElems);
```

Comment:
Unrelated with this PR, but I think how should consolidate how we deal with all the `data-hypothesis-[...]` attributes within the same file. So unifying `annotation-count.js` with `sidebar-trigger.js` (by the way, `sidebar-trigger.js` doesn't remove the event handlers correctly)

Reply:
I agree. It would make sense to combine the code for all these elements into one module.